tools/dev/run_dev_server.sh

```diff
@@ -20,11 +20,18 @@ export CLUSTER_HOSTNAME=${CLUSTER_HOSTNAME:-"weaviate-0"}
 export GPT4ALL_INFERENCE_API="http://localhost:8010"
 export DISABLE_TELEMETRY=true # disable telemetry for local development
 
+# inject build info into binaries.
+GIT_REVISION=$(git rev-parse --short HEAD)
+GIT_BRANCH=$(git rev-parse --abbrev-ref HEAD)
+
+VERSION="v$(jq -r '.info.version' openapi-specs/schema.json)"
```

Comment:
in configure_api.go we have a method called `parseVersionFromSwaggerSpec()` that parses the current version, this line is not needed

Reply:
I took inspiration from [here. ](https://github.com/weaviate/weaviate/blob/main/ci/push_docker.sh#L23). May be not needed for `run_dev_server`. happy to remove if you can confirm :)